python/lsst/obs/base/formatters/fitsExposure.py

```diff
@@ -107,7 +107,7 @@ def readMetadata(self):
         """
         # Do not use ExposureFitsReader.readMetadata because that strips
         # out lots of headers and there is no way to recover them
-        from lsst.afw.image import readMetadata
+        from lsst.afw.fits import readMetadata
```

Comment:
Move the import to the top of the file?

Reply:
I think this was placed where it is when it was in daf_butler, which did not have an explicit afw dependency, but I'm not positive. I think I'd rather leave it for now, unless @timj has some more feedback on it?

Reply:
I think @parejkoj is correct. We were deferring imports so we didn't trigger afw -- it should be fine to move it now.